pkg/controller/controller.go

```diff
@@ -256,3 +248,26 @@ func (c *Controller) kubeNodesInformer(stopCh <-chan struct{}, wg *sync.WaitGrou
 
 	c.nodesInformer.Run(stopCh)
 }
+
+func (c *Controller) getEffectiveNamespace(namespaceFromEnvironment, namespaceFromConfigMap string) string {
+
+	namespace := util.Coalesce(namespaceFromEnvironment, util.Coalesce(namespaceFromConfigMap, spec.GetOperatorNamespace()))
+
+	if namespace == "*" {
+
+		namespace = v1.NamespaceAll
+		c.logger.Infof("Listening to all namespaces")
+
+	} else {
+
+		if _, err := c.KubeClient.Namespaces().Get(namespace, metav1.GetOptions{}); err != nil {
+			// the namespace may be created manually at runtime
+			c.logger.Warnf("Could not find the watched namespace %q", namespace)
```

Comment:
Does it make sense to continue knowing that the namespace we are asked to listen to doesn't exist?

Reply:
I am unsure about that. For now it makes sense only for manual testing/experimentation

Reply:
then I think we can replace it back with Fatalf, which would give you the visible outcome for manual testing and will avoid running no-op operators for the production use case in case someone made a mistake in the configuration.